plugins/generator-1.14.4/forge-1.14.4/procedures/spawn_gem.java.ftl

```diff
@@ -1,6 +1,9 @@
 <#include "mcitems.ftl">
 if(world instanceof World && !world.getWorld().isRemote) {
 	ItemEntity entityToSpawn=new ItemEntity(world.getWorld(), ${input$x}, ${input$y}, ${input$z}, ${mappedMCItemToItemStackCode(input$block, 1)});
-	entityToSpawn.setPickupDelay(10);
+	entityToSpawn.setPickupDelay(${input$pickUpDelay!10});
+    <#if (field$despawn!false)?lower_case == "false">

```

Comment:
Why is the default value false, but default value in field is checked, so true? Same for 1.15.2 variant?

Reply:
No, when the field is checked, it mean that the item does despawn. Except that the code after the `if` deals with `noDespawn` so it's false if the item despawns (so if the field is checked)

Reply:
The default value here is false, in UI, it is true, so this is not OK.

In UI with existing block, tick is checked, but then generator assumes false default value when it is true (checked)

Reply:
Existing procedures will get setNoDespawn, but this was not there before, also it does not match with UI. Please test yourself and fix.

Reply:
It's done. I set default value to false so UI and code matches.